.github/CODEOWNERS

```diff
@@ -106,5 +106,16 @@ clang/test/AST/Interp/ @tbaederr
 # MLIR Sparsifier.
 /mlir/**/*SparseTensor*/ @aartbik @PeimingLiu @yinying-lisa-li @matthias-springer
 
+# MLIR NVGPU Dialect
+/mlir/**/NVGPU/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
```

Comment:
What's the difference between the last two lines?

Also can you compress the three lines into: `mlir/**/NVGPU*/` ?